.travis.yml

```diff
@@ -16,6 +16,18 @@ before_script:
 
 cache:
   bundler: true
+  directories:
```

Comment:
Do we need to be caching `$HOME/docker`? I don't see anything getting put there.

Reply:
Certainly not. It propagated from another changeset which should be removed. Will fix in a moment.